README.md

```diff
@@ -35,7 +35,7 @@
 
 * View all containers:
 
-      kubectl get pods
+      kubectl get pods / kubectl get pods --all-namespaces
```

Comment:
additionally `--namespace rucio` for rucio only services?

Reply:
It --namespace rucio is not working. Instead I can use `kubectl get namespaces` to get all the namespaces. I saw the specific rucio server is running in the default namespace by running `kubectl get pods --all-namespaces`. Then to specifically get the namespaces where rucio server is running I am running `kubectl get pod -n default` where I can simply see the status of my rucio server. It is actually working for me instead of writing only `kubectl get pods`. Thanks!  
![image](https://user-images.githubusercontent.com/77008026/224012766-28a4402e-c565-432e-b0cd-dd9b14b2e05a.png)